truncate/run.go

```diff
@@ -0,0 +1,164 @@
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package truncate
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"cloud.google.com/go/spanner"
+	"github.com/gosuri/uiprogress"
+)
+
+func Run(ctx context.Context, projectID, instanceID, databaseID string, quiet bool, out io.Writer, targetTables, excludeTables []string) error {
```

Comment:
Could you consider adding a func comment here? https://tip.golang.org/doc/comment#func

Example: 
```
// Run starts a routine to delete all rows from the specified database.
// If targetTables is not empty, it deletes from the specified tables.
// Otherwise it deletes from all tables in the database.
// If excludeTables is not empty, those tables are excluded from the deleted tables.
```